src/interface_adapter/logged_in/LoggedInState.java

```diff
@@ -1,10 +1,14 @@
 package interface_adapter.logged_in;
 
+import entities.auth.User;
+
 public class LoggedInState {
   private String username = "";
+  User user;
 
   public LoggedInState(LoggedInState copy) {
```

Comment:
Is this constructor ever called?

Reply:
No but it may be when I add the Memory DAO